pages/common/git-mv.md

```diff
@@ -0,0 +1,11 @@
+# git mv
+
+> Move or rename files and update the git index.
+
+- Rename or move file and add it to next commit
+
+`git mv {{filename}} {{new_filename}}`
+
+- Rename or move file, even if target exists
```

Comment:
Yes, it's just a change in the phrasing, to make the message more explicit, but the message is the same :)